src/app/core/common-components/entity-field-edit/entity-field-edit.component.html

```diff
@@ -1,7 +1,10 @@
 <div
   *ngIf="_field && form && _field.editComponent; else displayComponent"
   class="flex-row"
-  [ngClass]="{ hidden: _field?.hideFromForm }"
+  [ngClass]="{
+    hidden: _field?.hideFromForm,
+    readonlyAfterSet: isReadonlyAfterSet,
```

Comment:
make this style conditional only when in edit mode

Reply:
I tried, but I'm unable to get the boolean value for edit mode, which I need to conditionally show the warning and apply styling in this component.

Reply:
`form.formGroup` gives access to the overall form and its state. I added a condition now.